english/decline/decline.md

```diff
@@ -0,0 +1,26 @@
+You stare at the countless paths ahead of you, each an unknown possibility of failure.
+
+Looking from path to path, you start to spin as your head does the same. With every possible action, a preview plays in your head of predicted negative outcome.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.